src/main/java/at/hannibal2/skyhanni/data/ItemClickData.kt

```diff
@@ -21,11 +23,14 @@ class ItemClickData {
             val position = packet.position.toLorenzVec()
             BlockClickEvent(ClickType.RIGHT_CLICK, position, packet.stack).postAndCatch()
         }
+        val itemInHand = Minecraft.getMinecraft().thePlayer.heldItem
```

Comment:
Any reason why you moved the variable outside the if block?
This could be problematic when we get other packets from the server while the player is null. (Yes, this rarely happens)